File: hrmsFe/src/app/model/getBlackListResponse.ts
import {GetJobSeekerResponse} from "./getJobSeekerResponse";
import {GetHumanResourceEntityResponse} from "./getHumanResourceEntityResponse";

export interface GetBlackListResponse {

  id: string,

  description: string,

  jobSeeker: GetJobSeekerResponse,

  humanResourceEntity: GetHumanResourceEntityResponse
}

File: hrms/src/main/java/com/obss/hrms/response/GetBlackListResponse.java
package com.obss.hrms.response;

import com.obss.hrms.entity.HumanResource;
import com.obss.hrms.entity.HumanResourceEntity;
import com.obss.hrms.entity.JobSeeker;

public record GetBlackListResponse(
        String id,
        String description,
        JobSeeker jobSeeker,
        HumanResourceEntity humanResourceEntity
) {
}
